test/requests/support/dependency_detector_test.rb

```diff
@@ -39,5 +39,10 @@ def test_detects_rails_if_both_rails_and_minitest_are_present
 
       assert_equal("rails", DependencyDetector.detected_test_library)
     end
+
+    def test_direct_dependency_returns_false_outside_of_bundle
+      File.expects(:exist?).with("#{Dir.pwd}/Gemfile").once.returns(false)
```

Comment:
This should change to `File.file?` since that's what's used inside Bundler: https://github.com/rubygems/bundler/blob/master/lib/bundler/shared_helpers.rb#L239-L243